akka-projection-grpc/src/main/scala/akka/projection/grpc/replication/scaladsl/Replication.scala

```diff
@@ -121,4 +121,53 @@ object Replication {
         topicMatcher.matches(env, settings.eventProducerSettings.topicTagPrefix))(replicatedBehaviorFactory)
   }
 
+  /**
+   * Called to bootstrap the entity on each edge cluster node. In edge mode all connections for replication
+   * comes from edge node to cloud. Cloud service needs to be configured with edge replication enabled through
+   * `ReplicationSettings.withEdgeReplication(true)`.
+   *
+   * Important: Note that this does not publish the endpoint, additional steps are needed!
+   */
+  // FIXME factory method name
+  // FIXME filtering is important here, to control what gets replicated to the edge
+  // FIXME a separate settings class because it differs enough?
+  // FIXME maybe not grpcClientSettings directly?
+  // FIXME replica id of the edge node should not be in the otherReplicas of cloud nodes, is documenting that enough?
+  // FIXME replica id should still be unique, is documenting that enough (tricky to do any checks)?
+  def grpcEdgeReplication[Command, Event, State](settings: ReplicationSettings[Command])(
+      replicatedBehaviorFactory: ReplicatedBehaviors[Command, Event, State] => Behavior[Command])(
+      implicit system: ActorSystem[_]): EdgeReplication[Command] = {
+    // FIXME types to protect against this instead of a runtime check
+    if (settings.otherReplicas.size != 1)
+      throw new IllegalArgumentException(
+        s"ReplicationSettings contains multiple ${settings.otherReplicas.size} replicas, for edge replication those should be exactly one, the replica to connect to")
```

Comment:
I think it should be allowed to connect to more than one, although one is the most common case. Could be needed for some kind of redundancy.

Somewhat related to #1070

Reply:
Changed to replicate to all listed